src/main/java/de/tum/in/www1/artemis/web/rest/open/PublicResultResource.java

```diff
@@ -79,7 +79,7 @@ public PublicResultResource(Optional<ContinuousIntegrationService> continuousInt
     @PostMapping("programming-exercises/new-result")
     @EnforceNothing
     public ResponseEntity<?> processNewProgrammingExerciseResult(@RequestHeader("Authorization") String token, @RequestBody Object requestBody) {
-        log.debug("Received result notify (NEW)");
+        log.info("Received result notify (NEW): {}", requestBody);
```

Comment:
Do you really want to print out the whole request body?
I'd agree with a info log for the previous implementation but if you spring the body, maybe it should stay a debug log to be only logged when necessary..

Reply:
This was only for temporal debugging on GitLab to see what the whole request body looks like